soccer/src/soccer/strategy/agent/position/robot_factory_position.cpp

```diff
@@ -16,8 +16,63 @@ RobotFactoryPosition::RobotFactoryPosition(int r_id) : Position(r_id) {
 
 std::optional<RobotIntent> RobotFactoryPosition::get_task(WorldState& world_state,
                                                           FieldDimensions& field_dimensions) {
-    // This is where the current_position can be reassigned based on the
-    // PlayState
+    // If keeper, make no changes
+    if (robot_id_ == 0) {
+        return current_position_->get_task(world_state, field_dimensions);
+    }
+
+    // TODO (Rishi and Jack): Make this synchronized across all robots to avoid race conditions
+
+    // Get sorted positions of all friendly robots
+    typedef struct std::pair<int, double> RobotPos;  // (robotId, yPosition)
```

Comment:
```suggestion
    using RobotPos = std::pair<int, double>;  // (robotId, yPosition)
```
note the GH Action warning here. just simpler syntax it's completely equivalent